shared/Microsoft.Extensions.StackTrace.Sources/StackFrame/PortablePdbReader.cs

```diff
@@ -0,0 +1,135 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using System.Reflection.Metadata;
+using System.Reflection.Metadata.Ecma335;
+using System.Reflection.PortableExecutable;
+
+namespace Microsoft.Extensions.StackTrace.Sources
```

Comment:
we use StackTraceHelper from our package in Hosting and Diagnostics and that's in the same namespace. Changing the ns for that type would be the troublesome part.